openmdao/drivers/test/test_scipy_optimizer.py

```diff
@@ -281,7 +281,7 @@ def test_simple_array_comp_SLSQP_scaler_unity_eq(self):
         prob.driver.options['optimizer'] = 'SLSQP'
         prob.driver.add_desvar('x', lower=-50.0, upper=50.0, scaler=np.array([1.0, 1.0]))
 
-        prob.driver.add_objective('o', scaler=np.array([1.0, 1.0]))
+        prob.driver.add_objective('o', scaler=1.0)
```

Comment:
I just noticed that this is the objective. Yeah, this looks like a cut and paste error, so your fix is correct.